### PR TITLE
Move geo to cabby subpackage

### PR DESCRIPTION
**Pull request by jasonbaldridge**
Moved geo to cabby/geo and updated imports. 
Also fixed some documentation and BUILD dependencies.

### CODE REVIEW

File: cabby/geo/geo_computation.py [hunk 1]
```diff
@@ -38,16 +45,21 @@
 def main(argv):
     del argv  # Unused.
 
-    # Compute graph over Manhattan.
+    print("Obtaining graph for Manhattan.")
```
Comment (sayalikulkarni) [inline cabby/geo/geo_computation.py:48]: nit: use a logger instead?

Comment (jasonbaldridge) [reply cabby/geo/geo_computation.py:48]: Added a TODO for tzufgoogle@ to do that in another PR.

**Comment by sayalikulkarni** [comment]
lgtm

**Comment by jasonbaldridge** [comment]
Great, thanks!

**Pull request merged**